### PR TITLE
export several files to zip file (#203)

### PR DESCRIPTION
**Pull request by schochastics**

### CODE REVIEW

**Comment by schochastics** [comment]
@chainsawriot please have a look if the logic I used makes sense to you

File: R/export_list.R [hunk 1]
```diff
@@ -76,14 +78,20 @@ function(
         }
         outfiles <- file
     }
-
+    if (is.na(archive_format$compress) & archive_format$file != "") {
```
Comment (chainsawriot) [inline R/export_list.R:81]: 1. Use `&&` (gp)

2. This won't write to new directory:

```r
## if archivea is not created
export_list(mylist, file = paste0("file_", 1:3, ".xlsx"), archive = "archivea")
```

Maybe check and create it first?

Comment (chainsawriot) [reply R/export_list.R:81]: Also, I suggest adding `normalizePath` to `outfiles` and `archive`. Windows cannot handle "~/" without `normalizePath`

```r
outfiles <- base::normalizePath(outfiles)
archive <- base::normalizePath(archive) ## if it is not ""
```

ropensci/readODS#157

**Review by chainsawriot — COMMENTED** [review]
Just two comments

**Comment by chainsawriot** [comment]
@schochastics Thanks for finding this out. I will open a new issue and fix that.

**Comment by schochastics** [comment]
@chainsawriot actually since export_list calls export for the default case, I will only implement the "new dir" feature for the archive

**Comment by chainsawriot** [comment]
@schochastics OK, just leave it like that and I will fix #347 

I think the expected output for `test_export_list.R:25:5` should be the `normalizedPath(archive)`

**Comment by chainsawriot** [comment]
@schochastics Great! Thank you!

**Pull request merged**